### ISSUE TITLE
Support LinguaLeo API 1.0.1

### ISSUE THREAD

**Issue by troggy** [troggy/anki-leo#40]
Today or yesterday Lingualeo changed the API they use. Old one still partially works, but paging is broken now leading to issues like #39.

**Comment by troggy** [comment]
newer API doesn't look solid (e.g. mess with `translations` and `combinedTranslation` attributes). I expect it to break again soon :(

**Comment by troggy** [comment]
New API returns more words than shown on the page 😃🤦‍♂️

```sh
$ cat ~/Downloads/lingualeo.csv | awk -F ';' '{print $1}' | sed s/\"//g | wc -l
    2140
```

Without duplicates:

```sh
$ cat ~/Downloads/lingualeo.csv | awk -F ';' '{print $1}' | sed s/\"//g | uniq | wc -l
    2140
```

On the page it is just 2111

**Comment by troggy** [comment]
Given that Linguleo is not capable to show all the words itself (see below), I would not trust the 2111 number and consider mine (2140) is correct

![2019-08-04 18-58-01 2019-08-04 18_59_16](https://user-images.githubusercontent.com/163447/62426161-ff65af00-b6e9-11e9-9f7e-41b782ed84c1.gif)

**Comment by SGmuwa** [comment]
> Given that Linguleo is not capable to show all the words itself (see below), I would not trust the 2111 number and consider mine (2140) is correct

May you push develop (not stable) branch to GitHub? 🕒

**Comment by troggy** [comment]
yeah, sure.. sorry, I'm out of capacity to finish this up as of now

Here is the branch: https://github.com/troggy/anki-leo/fix/api101

**Comment by troggy** [comment]
closed via #41 . New version published 3.3.5
